dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -751,7 +751,7 @@ public User updateLoggedInUserMetadata(@ApiParam(hidden = true)@Parameter(hidden
         if (source.equals(TokenType.GOOGLE_COM)) {
             updateGoogleAccessToken(user.getId());
         }
-        dbuser.updateUserMetadata(tokenDAO, source);
+        dbuser.updateUserMetadata(tokenDAO, source, true);
```

Comment:
Given that you explicitly specifying the parameter here that it should throw an exception, does anybody still call the method on line 235 of User.java? If not then delete it.

Reply:
There are still uses of the method in line 235. The method in line 235 is only used when there is no source specified, but in this case, a source exists so `dbuser` is calling directly on `updateUserMetadata` on line 250. As for the test, I don't see any existing tests for `updateUserMetadata`. I can probably add a new test for the endpoint, but I don't believe I can finish it before the end of the sprint.